tests/ui/parser/obsolete-auto-trait-syntax.rs

```diff
@@ -0,0 +1,14 @@
+trait Trait1 {}
```

Comment:
can you make this a revision test to make this fully independent of recovery, making sure that it passes for the `#[cfg(not_enabled)]` case?